.circleci/config.yml

```diff
@@ -66,6 +66,19 @@ jobs:
       - run:
           command: |
             poetry run flake8
+  deploy:
+    docker:
+      - image: circleci/python:3.7.3
+    steps:
+      - add_ssh_keys
+      - checkout
+      - run: make check-has-release
+      - run: make install-deploy-deps
+      - run: make prepare-release
+      - run: poetry build
+      - run: make create-github-release
+      - run: make publish-changes
+      - run: poetry publish --username $PYPI_USERNAME --password $PYPI_PASSWORD
```

Comment:
doesn't look like we can publish using a token, so I guess we should create a new user for this :)